resources/scripts/components/server/Console.tsx

```diff
@@ -55,6 +58,9 @@ export default () => {
     const TERMINAL_PRELUDE = '\u001b[1m\u001b[33mcontainer@pterodactyl~ \u001b[0m';
     const ref = useRef<HTMLDivElement>(null);
     const terminal = useMemo(() => new Terminal({ ...terminalProps }), []);
+    const fitAddon = new FitAddon();
+    const searchAddon = new SearchAddon();
+    const searchAddonBar = new SearchBarAddon({ searchAddon });
```

Comment:
should be called `searchBarAddon`

Reply:
Fixed.

Reply:
Correction, it cannot be called `searchBarAddon` as that's also the name of the package. I've named it SearchBar